src/cbmc/cbmc_parse_options.cpp

```diff
@@ -106,6 +109,25 @@ void cbmc_parse_optionst::set_default_options(optionst &options)
   options.set_option("depth", UINT32_MAX);
 }
 
+void cbmc_parse_optionst::set_default_analysis_flags(optionst &options)
+{
+  // Checks enabled by default in v6.0+.
+  options.set_option("bounds-check", true);
+  options.set_option("pointer-check", true);
+  options.set_option("pointer-primitive-check", true);
+  options.set_option("div-by-zero-check", true);
+  options.set_option("signed-overflow-check", true);
+  options.set_option("undefined-shift-check", true);
```

Comment:
Aren't all of these then re-set later depending on the `[no-]standard-checks` option?

Reply:
This code is called *depending* on whether the `no-standard-checks` is set.

Reply:
Aren't they called on both line 140 and line 337?

Reply:
Ah, I see what you mean now. Yeah, that has been eliminated now :) Good catch, it was an artefact of a previous design that set the default options unconditionally, and then if the `--no-standard-checks` flag was present it was just overriding it.

Now we've moved to a design that sets or doesn't set based on the flag being present.